app.py

```diff
@@ -1,24 +1,24 @@
 from flask import Flask, render_template
 import os
 import simplejson as json
+import dateutil
 import bokeh.resources
 import bokeh.plotting
 import bokeh.embed
 from elasticsearch import Elasticsearch
 from textblob import TextBlob
+import pandas as pd
+import nltk
 import es_client
-
-app = Flask(__name__)
+import plotting
 
 
-@app.route('/')
-def index():
-    return render_template('index.html', resources=bokeh.resources.CDN.render())
+nltk.download('punkt')
+app = Flask(__name__)
 
 
-@app.route('/get_tweets.json', methods=['GET'])
-def get_tweets():
-    if 'PRODUCTION' in os.environ:
+def _get_env_params():
+    if 'APP_ENV' in os.environ and os.environ['APP_ENV'] == 'PRODUCTION':
```

Comment:
minor nitpick: I think you can provide a default value with `os.getenv('APP_ENV', 'default_value_of_something')` and so you don't need that `if 'APP_ENV' in os.environ` check

Reply:
Ahh yes, I should have known! I will most definitely use that in the next commit.

Reply:
Ready!